client/src/modules/list/model/actions.js

```diff
@@ -201,16 +201,20 @@ export const fetchListData = listId => dispatch =>
       throw err;
     });
 
-export const createList = data => dispatch =>
-  postData('/api/lists/create', data)
+export const createList = data => dispatch => {
+  const { type } = data;
+
+  return postData('/api/lists/create', data)
     .then(resp => resp.json())
     .then(json => {
       dispatch(createListSuccess(json));
       createNotificationWithTimeout(dispatch, NotificationType.SUCCESS, {
         notificationId: 'list.actions.create-list',
         data: data.name
       });
-      socket.emit(ListActionTypes.CREATE_SUCCESS, { json });
+      if (type === ListType.SHARED) {
+        socket.emit(ListActionTypes.CREATE_SUCCESS, json);
+      }
```

Comment:
what about LIMITED lists??

Reply:
limited lists are visible only for the creator so we don't have to send it to other users. If we add someone to limited list the will get this list via socket. Only shared lists are visible for all cohort members and need to be sent via sockets.